### ISSUE TITLE
[Newgrounds] youtube_dl.utils.ExtractorError: requested format not available

### ISSUE THREAD

**Issue by bruwalfas** [rg3/youtube-dl#12437]
- [*] I've **verified** and **I assure** that I'm running youtube-dl **2017.03.10**

### Before submitting an *issue* make sure you have:
- [*] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [*] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [*] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

```

$ youtube-dl -v http://www.newgrounds.com/portal/view/689710
[debug] System config: []
[debug] User config: ['-o', '%(title)s.%(ext)s', '-f', 'bestvideo[height<=1080][ext=mp4]+bestaudio/best[height<=720]']
[debug] Custom config: []
[debug] Command-line args: ['-v', 'http://www.newgrounds.com/portal/view/689710']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2017.03.10
[debug] Python version 3.4.4 - Windows-10-10.0.14393
[debug] exe versions: ffmpeg 3.2.2, ffprobe 3.2.2, rtmpdump 2.3
[debug] Proxy map: {}
[Newgrounds] 689710: Downloading webpage
WARNING: unable to extract uploader; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
ERROR: requested format not available
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpj3l3ckzu\build\youtube_dl\YoutubeDL.py", line 772, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpj3l3ckzu\build\youtube_dl\YoutubeDL.py", line 826, in process_ie_result
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpj3l3ckzu\build\youtube_dl\YoutubeDL.py", line 1519, in process_video_result
youtube_dl.utils.ExtractorError: requested format not available
```
http://www.newgrounds.com/portal/view/689710
---

Basically I cannot download this Newgrounds video in my computer. The weird thing is that I can perfectly watch it via stream.

**Comment by dstftw** [comment]
The format you are requesting with your config `-f bestvideo[height<=1080][ext=mp4]+bestaudio/best[height<=720]` does not exist.
